test/routes/variantText/variantTexts.test.js

```diff
@@ -0,0 +1,329 @@
+const HTTP_STATUS = require('http-status-codes');
+const supertest = require('supertest');
+const getPort = require('get-port');
+const db = require('../../../app/models');
+
+const CONFIG = require('../../../app/config');
+const {listen} = require('../../../app');
+
+CONFIG.set('env', 'test');
+const {username, password} = CONFIG.get('testing');
+
+const BASE_URI = '/api/variant-texts';
+
+jest.mock('../../../app/middleware/auth.js');
+
+let server;
+let request;
+
+const NON_ADMIN_GROUP = 'NON AUTHORIZED GROUP';
+const VARIANT_EDIT_ACCESS = 'variant-text edit access';
+
+const CREATE_DATA = {
+  text: '<p>sample text</p>',
+  variantName: 'variant name',
+  variantGkbId: 'v_gkb_id',
+  cancerType: 'cancer type',
+  cancerTypeGkbId: 'ct_gkb_id',
+};
+
+const UPLOAD_DATA = {
```

Comment:
these are the same, are both needed?

Reply:
CREATE_DATA and UPLOAD_DATA are both modified in line 94 to 98 to be different, since uploading requires an Ident, and creating directly an ID

Reply:
understood